src/Proto3/Suite/DotProto/AST.hs

```diff
@@ -396,13 +397,13 @@ instance Arbitrary DotProtoField where
     dotProtoFieldComment <- pure mempty
     return (DotProtoField {..})
 
-data DotProtoReservedField
+data DotProtoReserved
```

Comment:
Revert this name change back to `DotProtoReservedField`.

Reply:
Done in c78a6b5